*** AgendaByDayAdapter.smali.orig	Sun Apr 21 17:53:49 2013
--- AgendaByDayAdapter.smali	Sun Apr 21 17:55:41 2013
***************
*** 2373,2379 ****
  
      new-array v7, v7, [Ljava/lang/Object;
  
!     const/4 v8, 0x0
  
      iget v10, v14, Landroid/text/format/Time;->month:I
  
--- 2373,2379 ----
  
      new-array v7, v7, [Ljava/lang/Object;
  
!     const/4 v8, 0x1
  
      iget v10, v14, Landroid/text/format/Time;->month:I
  
***************
*** 2385,2391 ****
  
      aput-object v10, v7, v8
  
!     const/4 v8, 0x1
  
      iget v10, v14, Landroid/text/format/Time;->monthDay:I
  
--- 2385,2391 ----
  
      aput-object v10, v7, v8
  
!     const/4 v8, 0x0
  
      iget v10, v14, Landroid/text/format/Time;->monthDay:I
  
